app/views/posts/show.html.erb

```diff
@@ -1 +1,14 @@
-<h1>Show specific post for a given user</h1>
\ No newline at end of file
+<div class="container">
+    <div class="post-container">
+      <h4><%= @post.title %> by <%= @post.author.name %></h4>
+      <p><%= @post.text %></p>
+
+      <div class="comments-container">
+        <% @post.comments.each do |comment| %>
+          <p><%= comment.author.name %>: <%= comment.text %></p>
+        <% end %>
+      </div>
+    </div>
+</div>
```

Comment:
![image](https://github.com/kkolade/rails_blog_app/assets/93039359/4d00eaea-e57b-4efd-8884-96664bf8315b)


- [ ] It would be nice if you show the comments_counter and likes_counter here to make your design look similar to the given wireframe :+1: